src/main/java/me/lokka30/levelledmobs/listeners/EntityDeathListener.java

```diff
@@ -16,10 +23,8 @@
 import org.bukkit.inventory.ItemStack;
 import org.jetbrains.annotations.NotNull;
 
-import java.util.Arrays;
-import java.util.HashSet;
-import java.util.LinkedList;
-import java.util.List;
+import java.sql.Timestamp;
```

Comment:
Did we mean to import an SQL class? I think there are other ones like `Instant`, they are possibly better. If the SQL one works then that's fine

Reply:
I thought using sql class is fine  😃

Reply:
Should be all good :+1: :smile: